src/UnityRoslynUpdater.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net9.0-windows</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
I think this may lead to build warnings in the code using the registry APIs. You'll probably need to split that logic out into a separate function annotated with `[SupportedOSPlatform("windows")]` and then call it under an `if (OperatingSystem.IsWindows())` branch.